webapp-django/crashstats/base/templatetags/jinja_helpers.py

```diff
@@ -76,7 +76,7 @@ def change_query_string(context, **kwargs):
         base = ''
     else:
         base = context['request'].META['PATH_INFO']
-    qs = cgi.parse_qs(context['request'].META['QUERY_STRING'])
+    qs = urlparse.parse_qs(context['request'].META['QUERY_STRING'])
```

Comment:
This was deprecated, see https://docs.python.org/2/library/cgi.html#cgi.parse_qs